packages/admin/blocks-admin/src/blocks/helpers/composeBlocks/composeBlocks.tsx

```diff
@@ -191,6 +191,16 @@ export function composeBlocks<C extends CompositeBlocksConfig>(compositeBlocks:
 
                 return result;
             },
+            resolveDependencyRoute: (state, jsonPath) => {
```

Comment:
I believe createCompositeBlock needs its own implementation, especially as some blocks can be set as `nested: true`

Reply:
I feel like I don't know enough about the block system 😅 Let's talk about that next week.

Reply:
@johnnyomair Now I implemented it correctly for composite blocks